src/components/common/TmBalance.vue

```diff
@@ -172,10 +172,9 @@
           </div>
```

Comment:
@Bitcoinera look here for an implementation of our learning from the last 1:1

Reply:
ok! I will!

Reply:
It is already done, master 🙏

Reply:
:P

Reply:
lol, it was already done. just wanted you to checkout my implementation